### PR TITLE
Update feature manifest to add IBM-Maven-Dependency attribute.

### PR DESCRIPTION
**Pull request by alvinso**
Added IBM-Maven-Dependency attribute with maven coordinates to feature manifest

### CODE REVIEW

**Comment by alvinso** [comment]
link to the build

https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_Qh9zNhGqEeiStsw2He7iFw&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

File: dev/com.ibm.websphere.appserver.jaxws-2.2/com.ibm.websphere.appserver.jaxws-2.2.feature [hunk 1]
```diff
@@ -63,4 +63,4 @@ Subsystem-Name: Java Web Services 2.2
  bin/jaxws/tools/ws-wsgen.jar
 kind=ga
 edition=base
-IBM-Maven-Dependency: javax.xml.ws:jaxws-api:2.3
\ No newline at end of file
+IBM-Maven-Dependency: javax.xml.ws:jaxws-api:2.2
```
Comment (patricktiu) [inline dev/com.ibm.websphere.appserver.jaxws-2.2/com.ibm.websphere.appserver.jaxws-2.2.feature]: Base on https://repo.maven.apache.org/maven2/javax/javaee-api/7.0/javaee-api-7.0.pom, we should use javax.xml.ws:jaxws-api:2.2.8

File: dev/com.ibm.websphere.appserver.jsp-2.2/com.ibm.websphere.appserver.jsp-2.2.feature [hunk 2]
```diff
@@ -49,5 +49,5 @@ Subsystem-Name: JavaServer Pages 2.2
 -files=dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.jsp_1.0-javadoc.zip
 kind=ga
 edition=core
-IBM-Maven-Dependency: javax.servlet.jsp:javax.servlet.jsp-api:2.3.1, \
- javax.el:el-api:3.0.0
\ No newline at end of file
+IBM-Maven-Dependency: javax.servlet.jsp:javax.servlet.jsp-api:2.2.1, \
+ javax.el:el-api:2.2
```
Comment (patricktiu) [inline dev/com.ibm.websphere.appserver.jsp-2.2/com.ibm.websphere.appserver.jsp-2.2.feature]: You can remove javax.el:el-api:2.2 or chang eit to javax.el:el-api:2.2.1 (see https://mvnrepository.com/artifact/javax.servlet.jsp/javax.servlet.jsp-api/2.2.1)

**Comment by alvinso** [comment]
It is not needed anymore.

**Pull request closed without merging**